tests/std/tests/P0896R4_ranges_alg_sample/test.cpp

```diff
@@ -0,0 +1,83 @@
+// sampleright (c) Microsoft Corporation.

```

Comment:
Damaged banner.

Reply:
You will never no where I *copy*`d it from